### PR TITLE
Add _commit_message module

### PR DESCRIPTION
**Pull request by saicheems**
Contains a function which builds a nice, standard commit message for an
autogenerated update.

### CODE REVIEW

File: server/tasks/_commit_message.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+"""Contains a function which builds a nice commit message."""
```
Comment (vchudnov-g) [inline server/tasks/_commit_message.py]: License comment in all files.

File: server/tasks/_commit_message.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+from datetime import date
+
+
+def build(added, deleted, updated, subject=None):
+    """Returns a nice commit message.
```
Comment (vchudnov-g) [inline server/tasks/_commit_message.py]: Here and above, could we use something more descriptive than "nice"? "Comprehensive", "detailed", "complete", "consistent",....

File: server/tests/test__commit_message.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+def test_build_subject():
+    commitmsg = _commit_message.build(None, None, None, subject='X')
+    assert commitmsg == 'X'
+
+
+def test_build_added_subject():
+    added = {'foo:v1'}
+    commitmsg = _commit_message.build(added, None, None, subject='X')
+    assert commitmsg == 'X\n\nAdd:\n- foo:v1'
```
Comment (vchudnov-g) [inline server/tests/test__commit_message.py]: For readability here and below, consider using pseudo-multiline strings a la
```
assert commitmsg == ("X\n"
                     "\n"
                     "Add:\n"
                     "- foo:v1")
```

File: server/tasks/_commit_message.py (hunk 1 above)
Comment (saicheems) [reply server/tasks/_commit_message.py]: Done

File: server/tasks/_commit_message.py (hunk 2 above)
Comment (saicheems) [reply server/tasks/_commit_message.py]: Changed to "detailed"

File: server/tests/test__commit_message.py (hunk 3 above)
Comment (saicheems) [reply server/tests/test__commit_message.py]: Done

**Pull request merged**